flutter/ios/sentry_flutter/Package.swift

```diff
@@ -0,0 +1,36 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "sentry_flutter",
+    platforms: [
+        .iOS("12.0"),
+        .macOS("10.13")
+    ],
+    products: [
+        .library(name: "sentry-flutter", targets: ["sentry_flutter", "sentry_flutter_swift"])
+    ],
+    dependencies: [
+      .package(url: "https://github.com/getsentry/sentry-cocoa", from: "8.36.0"),
```

Comment:
do we need to adjust `update-cocoa.sh` for this?

Reply:
Yeah, we also need to implmenet the version update for the SPM file!